pkgs/applications/networking/cluster/terraform-providers/default.nix

```diff
@@ -15,9 +15,16 @@ let
       src = fetchFromGitHub {
         inherit owner repo rev sha256;
       };
-      # Terraform allow checking the provider versions, but this breaks
-      # if the versions are not provided via file paths.
-      postBuild = "mv go/bin/${repo}{,_v${version}}";
+      postBuild = ''
+        # Terraform allow checking the provider versions, but this breaks
+        # if the versions are not provided via file paths.
+        mv go/bin/${repo}{,_v${version}}
+
+        # Place provider in Terraform 0.13 search location
+        TF_13_PROVIDER_PATH=go/bin/${data.provider-source-address}/${version}/$(go env GOARCH)/
+        mkdir -p $TF_13_PROVIDER_PATH
+        ln -s go/bin/${repo}_v${version} $TF_13_PROVIDER_PATH
```

Comment:
I went with the symlink approach to avoid making two copies of the binary, but it doesn't work.

I also question whether it's `toDrv` responsibility to make the binary available in the correct location as opposed to only building. I'm open to recommendations :upside_down_face:.

Reply:
A wrapper script that execs the providers should be a good alternative. It's the approach we are taking in https://github.com/numtide/generate-terraform-provider-shim to download providers on demand and it works fine there.